README.md

```diff
@@ -101,12 +101,15 @@ site=https://realm.zulipchat.com
 theme=default
 # Autohide defaults to 'no_autohide', but can be set to 'autohide' to hide the left & right panels except when focused.
 autohide=autohide
-# Footlinks default to 'enabled', but can be set to 'disabled' to hide footlinks.
-footlinks=disabled
+# Maximum footlinks to be shown, defaults to 3, but can be set to any value>=0
+maximum-footlinks=3
 # Notify defaults to 'disabled', but can be set to 'enabled' to display notifications (see next section).
 notify=enabled
 # Color depth defaults to 256, but can be set to 1 (for monochrome) or 16.
 color-depth=256
+# Footlinks default to 'enabled', but can be set to 'disabled' to hide footlinks.
+# Cannot be used along side maximum-footlinks
+footlinks=disabled
```

Comment:
I'd leave this in the same location, with the new option right below it, with something like "if you need more flexibility...".